index.bs

```diff
@@ -2116,7 +2118,9 @@ When verifying a given {{PublicKeyCredential}} structure (|credential|) as part
 10. Using the credential public key looked up in step 1, verify that |sig| is a valid signature over the binary concatenation of
     |aData| and |hash|.
 
-11. If all the above steps are successful, continue with the authentication ceremony as appropriate. Otherwise, fail the
+11. Verify that the signature counter value is either (a) greater than the value stored or (b) it is equal to the value stored and the value stored is 0. 
```

Comment:
There needs to be an explanation of why we verify the signature counter. We should explain that if we verify the value is greater, that means the authenticator isn't cloned. If the value is 0, we don't know if the authenticator is cloned because of constraint by authenticators.

Reply:
Done.